app/views/pages/accessibility_statement.html.erb

```diff
@@ -0,0 +1,182 @@
+      the non-compliances listed below.
+    </p>
+
+    <ul class="govuk-list gove-uklist--bullet">
+      <li>
+        <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html" class="govuk-link">
+          2.4.2 – Page Titled
+        </a>
+      </li>
+      <li>
+        <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-suggestions.html" class="govuk-link">
+          3.3.3 – Error Suggestion
+        </a>
+      </li>
+      <li>
+        <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html" class="govuk-link">
+          4.1.1 – Parsing
+        </a>
+      </li>
+    </ul>
+
+    <h2 class="govuk-heading-l">Non accessible content</h2>
+
+    <p class="govuk-body">
+      The content listed below is non-accessible for the following reasons.
+    </p>
+
+    <ul class="govuk-list gove-uklist--bullet">
+      <li>
+        Page titles do not follow accessibility guidelines
+      </li>
+      <li>
+        Some error messages are inconsistently formatted.
+      </li>
+      <li>
+        Auto complete for address may not function correctly
+      </li>
+    </ul>
+
+    <h2 class="govuk-heading-l">Non compliance with the accessibility regulations</h2>
+
+    <p class="govuk-body">
+      Page titles are not clear an descriptive. They all have the same content.  This doesn't meet WCAG 2.1 success
+      criterion 2.4.2 (page titled).
+    </p>
+
+    <p class="govuk-body">
+      We plan to amend the page titles by December 2020. When we create new pages we will be sure to use descriptive
+      page titles.
+    </p>
+
+    <p class="govuk-body">
+      Some error messages are inconsistently formatted. There is a list of all errors at the top of each page, but some
+      pages to not provide guidance on how to resolve the error next to the relevant input.  This doesn't meet WCAG 2.1
+      success criterion 3.3.3 (error suggestion).
+    </p>
+
+    <p class="govuk-body">
+      We plan to amend the error messages by December 2020. When we create new pages we will make sure to use
+      consistently formatted error messages.
+    </p>
+
+    <p class="govuk-body">
+      Some pages return errors when parsed through a html validator. Although the pages should display correctly they
+      may have invlaid html. This doesn't meet WCAG 2.1 success criterion 4.1.1 (parsing).
+    </p>
+
+    <p class="govuk-body">
+      We plan to amend the pages to ensure they are valid htmlby December 2020. When we create new pages we will make
+      sure they validate.
+    </p>
+
+    <h2 class="govuk-heading-l">How we tested this website</h2>
+
+    <p class="govuk-body">
+      This website was last tested on 29 July 2019. The test was carried out internally.
+    </p>
+
+    <p class="govuk-body">
+      We tested:
+    </p>
+
+    <ul class="govuk-list gove-uklist--bullet">
```

Comment:
Typo: `gove-uklist--bullet`? (this is true everywhere in this file)

Reply:
Ok will fix that